lib/Pipfile

```diff
@@ -58,6 +58,7 @@ tzlocal = "*"
 validators = "*"
 watchdog = "*"
 # Only Python 3.x packages
+pydeck = "*"
```

Comment:
I think the comment above this doesn't make sense where it is. But it's also a useless comment anyway, so please remove it.

Same with the python 2.x comment below.